src/server.c

```diff
@@ -6185,6 +6247,17 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
                                   0, /* not a crash report */
                                   sections);
     }
+
+    if (dictFind(section_dict, "debug") != NULL) {
+        if (sections++) info = sdscat(info,"\r\n");
+        info = sdscatprintf(info,
+        "# Debug\r\n"
+        "eventloop_duration_aof_sum:%llu\r\n"
+        "eventloop_duration_cron_sum:%llu\r\n",
```

Comment:
re-posting this as a code comment text so for easier discussion (thread that can be resolved:

another interesting metric could be the number of commands per event loop cycle, specifically the max.
considering that we now intend to introduce an experimental section, maybe we can add `eventloop_cmd_per_cycle_max` and maybe re-add `eventloop_duration_max` to have them temporarily until histograms are present?
@madolson WDYT? is the non-`default` (and non-`all`) "Debug" section hidden enough to add these temporarily, or you still rather avoid them for now?

Reply:
@oranagra I think an explicit section that's not default or all is good enough.

Reply:
@oranagra I don't think we need to put temporary metrics for an unstable version, since eventually we will have histograms.

Reply:
> @madolson WDYT? is the non-default (and non-all) "Debug" section hidden enough to add these temporarily, or you still rather avoid them for now?

I'm okay with this.